src/app/components/modules/simplified-lunch/simplified-lunch.component.scss

```diff
@@ -0,0 +1,269 @@
+@import '../../../../styles/settings';
+
+/* --------------------------
+  Main Styles
+--------------------------- */
+.module-container{
+  width:100%;
+  height:100%;
+}
+.module-container[max-height~='500px']{
+  .not-main-entree{
+    display: none;
+  }
+  .lunch-category{
+    border-top: none;
+  }
+}
+.module-container[max-width~='500px']{
+  .not-main-entree{
+    display: none; 
+  }
+  .lunch-category{
+    border-top: none;
+  }
+}
+.module-container[max-height~='300px']{
+  .lunch-date{
+    display:none;
+  }
+  .lunch-category{
+    padding-top: 0rem;
+  }
+
+}
+
+.lunch-week .lunch-category{
+  border-top: 1px solid rgba(white, 0.5);
+
+}
```

Comment:
Unnecessary newline above, but we typically put a newline between root styles (so under `.module-container[max-width~='500px']{ ... }` but right above `.module-container[max-height~='300px']{ ...`)